lib/driver.js

```diff
@@ -270,7 +294,9 @@ class EspressoDriver extends BaseDriver {
         await presignApp(appPath);
       }
     }
-    return shouldResultAppPathBeCached ? {appPath: pathInCache} : false;
+
+    return shouldResultAppPathBeCached ?
```

Comment:
nested ternary conditions are hard to read. Can we add a simple if condition instead?